apis/datadoghq/common/const.go

```diff
@@ -81,13 +81,15 @@ const (
 	// DefaultHelmCheckConf default Helm Check ConfigMap name
 	DefaultHelmCheckConf string = "helm-check-config"
 
+	// DefaultAgentHealthPort default agent health port
+	DefaultAgentHealthPort int32 = 5555
```

Comment:
A TCP port number is 16 bits: https://en.wikipedia.org/wiki/Transmission_Control_Protocol#TCP_ports

```suggestion
	DefaultAgentHealthPort uint16 = 5555
```

Reply:
I ended up dropping this as `k8s.io/api/core/v1` `HTTPGetAction{}` specifically calls for a `int32`.
```
// IntOrString is a type that can hold an int32 or a string.  When used in
// JSON or YAML marshalling and unmarshalling, it produces or consumes the
// inner type.  This allows you to have, for example, a JSON field that can
// accept a name or number.
// TODO: Rename to Int32OrString
//
// +protobuf=true
// +protobuf.options.(gogoproto.goproto_stringer)=false
// +k8s:openapi-gen=true
type IntOrString struct {
	Type   Type   `protobuf:"varint,1,opt,name=type,casttype=Type"`
	IntVal int32  `protobuf:"varint,2,opt,name=intVal"`
	StrVal string `protobuf:"bytes,3,opt,name=strVal"`
}
```